tests/test_validation.py

```diff
@@ -1,20 +1,37 @@
 import logging
 import os
 import re
+from typing import Generator
+from unittest.mock import patch
 
 import pytest
+from pandas import Series, DataFrame, concat
 
 from schematic.models.metadata import MetadataModel
-from schematic.models.validate_attribute import GenerateError, ValidateAttribute
+from schematic.models.validate_attribute import (
+    GenerateError,
+    ValidateAttribute,
+    SetValidationOutput,
+    ValueValidationOutput,
+)
 from schematic.models.validate_manifest import ValidateManifest
 from schematic.schemas.data_model_graph import DataModelGraph, DataModelGraphExplorer
 from schematic.schemas.data_model_json_schema import DataModelJSONSchema
 from schematic.utils.validate_rules_utils import validation_rule_info
+import schematic.models.validate_attribute
+
+# pylint: disable=protected-access
 
 logging.basicConfig(level=logging.DEBUG)
 logger = logging.getLogger(__name__)
 
 
+@pytest.fixture(name="dmge")
+def DMGE(helpers) -> Generator[DataModelGraphExplorer, None, None]:
+    dmge = helpers.get_data_model_graph_explorer(path="example.model.jsonld")
+    yield dmge
```

Comment:
I moved this into `conftest.py` so you won't need to create this again. You should be able to delete this and your functions should test exactly the same.

Reply:
I see. There was a merge conflict and I added it back in.